daffodil-japi/src/main/scala/org/apache/daffodil/japi/packageprivate/Utils.scala

```diff
@@ -27,11 +27,11 @@ package org.apache.daffodil.japi.packageprivate
 import org.apache.daffodil.japi._
 import org.apache.daffodil.japi.logger._
 import org.apache.daffodil.japi.debugger._
-import org.apache.daffodil.api.{ ValidationMode => SValidationMode }
-import org.apache.daffodil.util.{ LogLevel => SLogLevel }
-import org.apache.daffodil.util.{ LogWriter => SLogWriter }
-import org.apache.daffodil.debugger.{ InteractiveDebugger => SInteractiveDebugger }
-import org.apache.daffodil.debugger.{ InteractiveDebuggerRunner => SInteractiveDebuggerRunner }
+import org.apache.daffodil.api.{ValidationMode => SValidationMode}
+import org.apache.daffodil.util.{LogLevel => SLogLevel}
+import org.apache.daffodil.util.{LogWriter => SLogWriter}
+import org.apache.daffodil.debugger.{InteractiveDebugger => SInteractiveDebugger}
+import org.apache.daffodil.debugger.{InteractiveDebuggerRunner => SInteractiveDebuggerRunner}
```

Comment:
Thanks.  I use scalafmt in all other projects, just been trying to avoid hitting organize imports in this one.  But that and format is such a habit it sneaks in sometime.  Ill take a look at that option for this project.

Reverted this change for now.